app.py

```diff
@@ -310,6 +316,51 @@ def save_identity(caller_identity):
     return jsonify({})
 
 
+# End Point which creates payout info next to identity
+@app.route('/v1/identities/<identity_url_param>/payout', methods=['POST'])
```

Comment:
Just a though: maybe singular resource would be simpler for current identity, i.e. `me` or `my_identity` - that way we wouldn't have to pass it and to validate whether `identity_url_param` matches current identity.
```suggestion
@app.route('/v1/my_identity/payout', methods=['POST'])
```

Reply:
I'd rather keep it restful. If we're working on an identity object, it should definitely stay on the /v1/identities.